giga_connectome/postprocess.py

```diff
@@ -136,21 +164,37 @@ def run_postprocessing_dataset(
             )
             connectomes[desc].append(correlation_matrix)
 
-            # dump to h5
-            flag = _set_file_flag(output_path)
-            with h5py.File(output_path, flag) as f:
+            # one output file per input file per atlas
+
+            # dump correlation_matrix to tsv
+            tsv_filename = connectome_path / utils.output_filename(
+                source_file=Path(img.filename).stem,
+                atlas=atlas["name"],
+                extension="tsv",
+                strategy=strategy["name"],
+                desc=desc,
+            )
+            utils.check_path(tsv_filename)
+            df = pd.DataFrame(correlation_matrix)
+            df.to_csv(tsv_filename, sep="\t", index=False)
+
+            # dump timeseries to h5
+            h5_filename = connectome_path / utils.output_filename(
+                source_file=Path(img.filename).stem,
+                atlas=atlas["name"],
+                extension="h5",
+                strategy=strategy["name"],
+                desc=desc,
+            )
```

Comment:
If we want to be fully BIDS compliant, shall we save time series to h5 as well?

Reply:
yeah am working on this

ideally maybe even in nifti, no?

Reply:
no nifty since we are not outputting seed based maps?
The only sensible nifti we can output from here is the denoised and smoothed nifti, but that would be out of scope IMO

Reply:
ok will dump that in tsv then